collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2100,6 +2100,10 @@ protected void onResume() {
 
         FormController formController = getFormController();
 
+        if (formController != null && !formEntryViewModel.isFormControllerSet()) {
```

Comment:
Is this not really likely on opening a form as `onResume` would usually happen before `loadingComplete` (as the latter only gets called after the background task is finished)?

Reply:
Prior to this crash I would have thought it impossible but I’m not so sure anymore. Figure it can’t hurt to confirm.

Reply:
Fair enough!

Reply:
I think we're going to see this happen after the log for `Calling onActivityResult from loadingComplete`. Because I think that branch is going to call `onActivityResult`, then return from `loadingComplete` and then `onResume` will be called.